src/main/java/com/springboot/blinkist/controller/AuthorController.java

```diff
@@ -0,0 +1,45 @@
+package com.springboot.blinkist.controller;
+
+import com.springboot.blinkist.dto.AuthorDTO;
+import com.springboot.blinkist.service.AuthorService;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/authors")
+public class AuthorController {
+
+    @Autowired
+    private AuthorService authorService;
+
+    @GetMapping
+    public List<AuthorDTO> getAuthors() {
+        return authorService.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public AuthorDTO getAuthor(@PathVariable int id) {
+        return authorService.findById(id);
+    }
+
+    @PostMapping
+    public String addAuthor(@RequestBody AuthorDTO authorDTO) {
+        authorService.save(authorDTO);
+        return "success!";
+    }
+
+    @PutMapping
+    public String updateAuthor(@RequestBody AuthorDTO authorDTO) {
+        authorService.save(authorDTO);
```

Comment:
handle exceptions if author id is not found (same for all controllers and services)